src/components/contents/Chat.js

```diff
@@ -0,0 +1,71 @@
+import React, {PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import Header from '../common/Header';
+import MessageListItem from '../common/MessageListItem';
+import * as messagesActions from '../../actions/messagesActions';
+import * as currentConversationActions from '../../actions/currentConversationActions';
+import * as contentActions from '../../actions/contentActions';
+
+export const Chat = ({ currentConversation, messages, actions }) => {
```

Comment:
Another trick here, you only use the action `switchContent` so, to avoid writing `actions.switchContent` you can use deep matching (http://es6-features.org/#ObjectMatchingDeepMatching)

` ({ currentConversation, messages, actions: { switchContent } })`